### ISSUE TITLE
prob

### ISSUE THREAD

**Issue by slighting666** [FingerRec/3DNet_Visualization#3]
How does the prob in the picture show up?

**Comment by FingerRec** [comment]
Hi, @slighting666 , sorry for late reply.

If you have label available, you can choose top-k value from the output logins.

**Comment by slighting666** [comment]
yes, I used the UCF-101 data set, but did not get the probability value you mentioned.
Can you give some suggestions, thank you!
@FingerRec

**Comment by FingerRec** [comment]
notice in the input script the option is --supervised if label available.

**Comment by slighting666** [comment]
> supervised

I use UCF-101 data set and set supervised. But something went wrong!
@FingerRec

**Comment by FingerRec** [comment]
add  .detach().cpu().numpy()  at last in line 152 of action_recognition.py
